plugins/inputs/mongodb/mongodb_data.go

```diff
@@ -86,6 +86,21 @@ var DefaultStats = map[string]string{
 	"connections_total_created": "TotalCreatedC",
 }
 
+var DefaultCommandsStats = map[string]string{
+	"delete_command_total":         "DeleteCommandTotal",
+	"delete_command_failed":        "DeleteCommandFailed",
+	"find_command_total":           "FindCommandTotal",
+	"find_command_failed":          "FindCommandFailed",
+	"findAndModify_command_total":  "FindAndModifyCommandTotal",
+	"findAndModify_command_failed": "FindAndModifyCommandFailed",
+	"getMore_command_total":        "GetMoreCommandTotal",
+	"getMore_command_failed":       "GetMoreCommandFailed",
```

Comment:
Can you further convert these to snake_case: `find_and_modify_command_total`

Reply:
Fixed in [92a6e0e3](https://github.com/influxdata/telegraf/pull/6905/commits/92a6e0e3fcd67cbcf6dc5b69d39b47d8d7e5b3a8)